src/Tribe/Cost_Utils.php

```diff
@@ -321,8 +321,14 @@ public function parse_cost_range( $costs, $max_decimals = null, $sort = true ) {
 
 		foreach ( $costs as &$cost ) {
 			// Get the required parts
+			$cost_wo_separators = $cost;
+
+			foreach ( $this->get_separators() as $separator ) {
```

Comment:
Non-essential, but we could simplify this a little and dispose of the `foreach()` loop by doing something like:

```php
$cost_wo_separators = str_replace( $this->get_separators(), '', $cost_wo_separators );
```